uniq-stacks.py

```diff
@@ -91,11 +91,11 @@ def invoke(self, argument, from_tty):
         sorter = lambda d: sorted(d.items(), key=lambda item: len(item[1]),
                 reverse=True)
 
-        gdb.write("\n== Printing {} unique stacks from {} threads\n\n".format(
+        gdb.write("\n== Printing {} unique stack(s) from {} thread(s)\n\n".format(
             len(uniq), len(traces)))
 
         for k, v in sorter(uniq):
-            gdb.write("Stack for thread ids {}\n".format(sorted(v)))
+            gdb.write("Unique stack common for {} thread(s); Thread id(s): {}\n".format(len(v), sorted(v)))
```

Comment:
Nit: this phrasing is a bit redundant, IMO. Would you mind removing the leading "Unique"?